src/main/java/core/basesyntax/impl/DataConverterImpl.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax.impl;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.DataConverter;
+import java.util.ArrayList;
+import java.util.List;
+
+public class DataConverterImpl implements DataConverter {
+    @Override
+    public List<FruitTransaction> convertToTransaction(List<String> data) {
+        List<FruitTransaction> transactions = new ArrayList<>();
+        boolean isFirstLine = true;
+        for (String line : data) {
+            if (isFirstLine) {
+                isFirstLine = false;
+                continue;
+            }
+            String[] parts = line.split(",");
+            if (parts.length != 3) {
+                continue;
+            }
+            FruitTransaction.Operation operation = FruitTransaction.Operation.fromCode(parts[0]);
```

Comment:
You should validate the operation obtained from `FruitTransaction.Operation.fromCode(parts[0])`. If the operation is not recognized, it would be better to throw an exception instead of adding an invalid `FruitTransaction` to the list.